sample_code/ep143/rev01/t.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import types
+
+import pytest
+
+
+@pytest.fixture
+def git_dir(tmpdir):
+    d = tmpdir.join('git')
+    subprocess.check_call(('git', 'init', str(d)))
+    subprocess.check_call(('git', 'commit', '--allow-empty', '-m', 'init'))
```

Comment:
oh yeah, I guess this is what I get for not running hte code -- this should have `git -C str(d) commit ...`

Reply:
Yeah, not a big deal, but it was hilarious, when I saw the `init` commits :D And decided to share the experience.

Reply:
probably ok to fix it here -- do you want to make it not-broken in case someone copy pastes?

Reply:
Let me know if you want me to add the `-C str(d)` part to the example or not.
Because if someone try the example, this will actually commit to the local `astpretty` repo :D